nixos/modules/services/hardware/monado.nix

```
@@ -0,0 +1,86 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib) mkDefault mkEnableOption mkIf mkOption mkPackageOption;
+
+  cfg = config.services.monado;
+
+in
+{
+  options.services.monado = {
+    enable = mkEnableOption "Monado wrapper and user service";
+
+    package = mkPackageOption pkgs "monado" { };
+
+    defaultRuntime = mkEnableOption "" // mkOption {
+      description = ''
+        When enabled, set Monado as the default OpenXR runtime on the system.
+      '';
+    };
+  };
```

Sorry, missed this the first time but why does this need a custom description?

```suggestion
    defaultRuntime = mkEnableOption "Monado as the default OpenXR runtime on the system";
  };
```